hack/jenkins/release_build_and_upload.sh

```diff
@@ -43,9 +43,10 @@ echo "Verifying ISO exists ..."
 make verify-iso
 
 # Build and upload
-env BUILD_IN_DOCKER=y \
+env BUILD_IN_DOCKER=y BUILD_IMAGE=golang:1.16beta1-buster \
```

Comment:
since we change in Makefile no need to change both places. so later we can revert only on file